src/server/auth-passport/local.ts

```diff
@@ -58,7 +58,7 @@ export function setupLocalAuthPassport() {
           uuidMatch,
           reqBody: req.body
         });
-        if (user.is_suspended) {
+        if (user?.is_suspended === true) {
```

Comment:
Question: Do we need the `=== true` here? If  `user` evaluates to not null then `is_suspended` also should?
```suggestion
        if (user?.is_suspended) {
```

Reply:
I agree ^^ Suggestion: Add it to the other places above and below too!